proposals/3/README.md

```diff
@@ -0,0 +1,183 @@
+    *   _kaniko-sa_
+        *   required
+        *   IAM roles:    
+            *   GKE - roles/iam.workloadIdentityUser, roles/storage.admin, roles/storage.objectAdmin, roles/storage.objectCreator
+            *   EKS - Covered by the tekton-bot Service Account.
+    *   _storage-sa_
+        *   required
+        *   IAM roles:    
+            *   GKE - roles/iam.workloadIdentityUser, storage.objects.[create|get|delete|update]
+            *   EKS - S3Access Policy attached to an IAM/role and a Kubernetes Service Account.
+    *   _tekton-sa_
+        *   required
+        *   IAM roles:    
+            *   GKE - roles/iam.workloadIdentityUser, roles/viewer 
+            *   EKS - IAM/role and IAM Policy with general permissions, Kubernetes Service Account.
+    *   _velero-sa_
+        *   optional, only created if the user wants to use Velero
+        *   IAM roles:    
+            *   GKE - roles/iam.workloadIdentityUser, roles/storage.admin, roles/storage.objectAdmin, roles/storage.objectCreator
+            *   EKS - S3Access Policy attached to an IAM/role and a Kubernetes Service Account.
+    *   _vault-sa_
+        *   required 
+        *   IAM roles:
+            *   GKE - roles/iam.workloadIdentityUser, roles/cloudkms.admin, roles/cloudkms.cryptoKeyEncrypterDecrypter, roles/storage.objectAdmin
+            *   EKS - ?           
+*   Storage buckets 
+      *   logs (required)
+      *   vault (required)
+      *   reports (optional, same bucket as logs if not provided)
+      *   repository (optional, used by bucketrepo if enabled)
+      *   backup (optional, used by Velero if enabled)
+*   Kryptographic keys
+
+Permissions needed by Kubernetes service accounts for the various Cloud APIs will be managed by [workload identity](https://cloud.google.com/kubernetes-engine/docs/how-to/workload-identity).
+
+Similar to the setup in [terraform-google-jx](https://github.com/jenkins-x/terraform-google-jx), the output of applying the Terraform plan will be a `jx-requirements.yaml` which can then be used as input for `jx boot`.
+
+The Terraform scripts and documentation will discuss and recommend how Terraform state should be managed, e.g. via remote storage in a cloud bucket.
+
+Initially, the versioning of the repository containing the Terraform scripts is not automatically linked to `jx` resp [jenkins-x-boot-config](https://github.com/jenkins-x/jenkins-x-boot-config) releases.
+Moving forward we can either automatically update the Terraform `jx-requirements.yaml` templates as new releases of the boot config become available or we integrate the install of Jenkins X into the Terraform script via the Helm provider. 
+
+
+### 2.2.3 Docs
+
+Ar part of this enhancement the documentation will get extended and will include sections for the following:
+
+*   Generic description of the cloud resources needed and their purpose
+*   Documentation on where to find the Terraform script and how to use them
+    *   Recommendations on 
+
```

Comment:
we should also:
* update the getting started guide(s) to ensure that any reference wot `jx create cluster` is replaced with the terraform commands as well as a link to more details
* update upgrade sections to highlight that upgrades may fail if new infrastructure is needed for a new feature. Details about this requirement will be mentioned in release notes